### ISSUE TITLE
Automatically open browser when running tests

### ISSUE THREAD

**Issue by ZmSama** [vitest-dev/vscode#145]
**Describe the bug**
This may not be a bug, but every click and run automatically opens the browser, which greatly affects the experience

**To Reproduce**
1. Click on the test icon in the sidebar
2. At this point, vscode will output a connection address on the command line
3. And then the default browser on this computer will automatically open
![image](https://user-images.githubusercontent.com/45474353/228716638-ab0a5b06-a1e9-4137-9bf2-6682f72bd7e3.png)
![image](https://user-images.githubusercontent.com/45474353/228716481-6a0fc0b3-0896-4913-bd49-0c33dd9d3a84.png)

**Environment**

(Paste info.txt content generated by [the example project](https://github.com/zxch3n/vitest-ext-basic-example))

- OS: windows10 22H2
- VSCode version: 1.76.1
- Vitest version: 0.29.8
- Vitest plugin version:3.1.0

**Comment by kamikazePT** [comment]
same here, it's annoying

**Comment by zxch3n** [comment]
Does this issue persist when running on the example project?

**Comment by phiter** [comment]
I found the solution for this issue. Vitest is picking the config from `vite.config.ts` and if you have `server.open` set to `true` it will open the browser.

Set it to `process.env.NODE_ENV === 'development'` and it should not open the browser anymore.

```ts
export default defineConfig({
    server: {
        port: 3000,
        open: process.env.NODE_ENV === 'development'
    },
```

**Comment by sheremet-va** [comment]
This should be fixed in pre-release 0.5.0 and higher. Note that the extension now requires Vitest 1.4.0 or higher.

We don't start a server anymore, so this is not a problem.
